packages/mui-material/src/Button/Button.d.ts

```diff
@@ -60,6 +60,11 @@ export interface ButtonOwnProps {
    * If defined, an `a` element will be used as the root node.
    */
   href?: string;
+   /**
+   * The target for link when the button is clicked.
+   * @default '_blank'
```

Comment:
```suggestion
```
We don't add `target="_blank"` by default.

Reply:
@siriwatknp So should I add `target="_blank"` as default value in the component or you want a different behaviour? Please let me know.
Thanks!

Reply:
@KumarNitin19 It's ok to not have a `@default` here like `href` above, just be sure to run `pnpm proptypes` to update the PropTypes and pass the CI

I would also suggest adding `target` to some of the tests in https://github.com/mui/material-ui/blob/master/packages/mui-material/src/Button/Button.spec.tsx as well!

Reply:
@mj12albert Okay I will add the test case and remove the default state.